src/modules/gnd_pos_control/gnd_pos_control_params.c

```diff
@@ -257,6 +257,32 @@ PARAM_DEFINE_FLOAT(GND_SPEED_TRIM, 3.0f);
  * @max 40
  * @decimal 1
  * @increment 0.5
- * @group FW TECS
+ * @group GND POS Control
  */
 PARAM_DEFINE_FLOAT(GND_SPEED_MAX, 10.0f);
+
+/**
+ * Distance from front axle to rear axle
+ *
+ *
+ * @unit m
+ * @min 0.0
+ * @decimal 3
+ * @increment 0.01
+ * @group GND POS Control
```

Comment:
This is the metadata that an end user sees in QGC. For example MC is `@group Multicopter Position Control`.

How about `Ground Rover Position Control` or even just `Rover Position Control` (needs to be consistently used everywhere)?